wallets/react-wallet-chat/src/utils/WalletConnectUtil.ts

```diff
@@ -21,6 +21,7 @@ export async function createSignClient() {
 export async function createChatClient() {
   chatClient = await ChatClient.init({
     logger: 'debug',
+    keyseverUrl: 'https://keys.walletconnect.com',
```

Comment:
Typo:

```suggestion
    keyserverUrl: 'https://keys.walletconnect.com',
```

Reply:
Good catch, this is actually a typo in chat client, fixed here: https://github.com/WalletConnect/chat-client-js/pull/34